common/obtain_token.py

```diff
@@ -0,0 +1,100 @@
+# This file generates a valid google session using credentials and a token
+
+import warnings
+
+import click
+import google.auth
+import google.auth.transport.requests
+from google.auth import compute_engine
+from google.auth import impersonated_credentials
+from google.auth.transport import requests  # type: ignore
+from google.auth.transport.requests import AuthorizedSession
+from google.oauth2 import service_account
+
+
+# If you want to use a human account or service account
+def get_creds_from_human_user():
```

Comment:
I think you also need some PyDocs on these methods and definitely refer to the methods in the Readme

Reply:
Good catch! Will do now.